Tests/Smoketest4.xctestplan

```diff
@@ -70,7 +70,6 @@
         "NavigationTest\/testWriteToChildPopupTab()",
         "NewTabSettingsTest",
         "NightModeTests",
-        "NoImageTests",
```

Comment:
If I'm not mistaken this Test is part of the Full Functional, not the Smoktest, so it should be ketp as disabled here... or if it is part of the Smoktest, it should be disabled from the Full Functional test suite.
No need to run the test as part of the two test suites, having it enabled in one would be enough

Reply:
i've noticed that on testrail this test is on both plans, so this is why i added it like this